src/client/java/teammates/client/scripts/DataMigrationForFeedbackQuestionsToQuestions.java

```diff
@@ -97,11 +109,15 @@ protected void doOperation() {
                                     e);
                 } catch (EntityAlreadyExistsException e) {
                     // ignore if a copy of the old question already exists
-                    System.out.println("new qn type already exists for question:" + old.getIdentificationString());
+                    System.out.println("New question type entity already exists for question:" + old.getIdentificationString());
                 }
             }
+            
+            if (i % 100 == 0) {
+                System.out.println("Migrating the " + i + "'th Question");
```

Comment:
Rather than printing the number every 100 questions, maybe you could prefix the printed statement of each iteration with the current index, e.g. 
`1. Will create question: qwerty`